[2024-10-03T18:15:24.563+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T18:15:24.574+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T12:45:07.842808+00:00 [queued]>
[2024-10-03T18:15:24.582+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T12:45:07.842808+00:00 [queued]>
[2024-10-03T18:15:24.582+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T18:15:24.615+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 12:45:07.842808+00:00
[2024-10-03T18:15:24.625+0530] {standard_task_runner.py:72} INFO - Started process 10928 to run task
[2024-10-03T18:15:24.631+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T12:45:07.842808+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmp_o4k7ad3']
[2024-10-03T18:15:24.637+0530] {standard_task_runner.py:105} INFO - Job 86: Subtask validate_data
[2024-10-03T18:15:24.708+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T12:45:07.842808+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T18:15:24.824+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T12:45:07.842808+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T12:45:07.842808+00:00'
[2024-10-03T18:15:24.825+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T18:15:24.908+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T18:15:24.911+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T18:15:24.996+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T18:15:25.165+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T18:15:25.165+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T18:15:25.166+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 2149.27it/s]
[2024-10-03T18:15:25.199+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 59.60it/s]  
[2024-10-03T18:15:25.199+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:00, 87.98it/s]
[2024-10-03T18:15:25.228+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 47.47it/s]
[2024-10-03T18:15:25.235+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 244.26it/s]
[2024-10-03T18:15:25.260+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 179.90it/s]
[2024-10-03T18:15:25.286+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 607.18it/s]
[2024-10-03T18:15:25.286+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 607.18it/s]
[2024-10-03T18:15:25.297+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 607.18it/s]
[2024-10-03T18:15:25.297+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 607.18it/s]
[2024-10-03T18:15:25.297+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 552.12it/s]
[2024-10-03T18:15:25.643+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T18:15:26.303+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T18:15:26.309+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T18:15:26.309+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T12:45:07.842808+00:00, execution_date=20241003T124507, start_date=20241003T124524, end_date=20241003T124526
[2024-10-03T18:15:26.375+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T18:15:26.404+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T18:15:26.422+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
